kernel/kernel-api/src/main/java/io/delta/kernel/Snapshot.java

```diff
@@ -36,6 +36,21 @@ public interface Snapshot {
    */
   long getVersion(Engine engine);
 
+  /**
+   * Get the timestamp (in milliseconds since the Unix epoch) of the latest commit of this snapshot.
+   * For an uninitialized snapshot, this returns -1.
```

Comment:
Can you specify what "uninitialized snapshot" means?

Reply:
This was just met copying + pasting the existing javadocs.

Let me think about this. We should probably just say "for an empty table"